WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt

```diff
@@ -45,7 +45,7 @@ sealed class MySiteCardAndItem(open val type: Type, open val activeQuickStartIte
         SINGLE_ACTION_CARD,
         JETPACK_FEATURE_CARD,
         JETPACK_SWITCH_CARD,
-        JETPACK_INSTALL_FULL_PLUGIN_CARD
+        JETPACK_INSTALL_FULL_PLUGIN_CARD,
```

Comment:
np: I guess you forgot to remove `,`.

Reply:
This started happening after I installed copolit. However I did check that the trailing comma passes checkstyle, lint, and detekt before I left them. Is there really a right way or wrong way or is it just a matter of personal preference? Not something worth falling on my sword for, so I'll jot it down on my list to pull them out later on.

Reply:
An extra comma fails in Java but not in Kotlin. There's no warning for it. Therefore, it's mostly a personal style preference, so feel free to ignore it.